pychunkedgraph/graph/subgraph.py

```diff
@@ -221,4 +231,13 @@ def _get_subgraph_multiple_nodes_threaded(
         )
         cur_nodes_children = dict(ChainMap(*cur_nodes_child_maps))
         subgraph.process_batch_of_children(cur_nodes_children)
+
+    if len(return_layers) == 1:
+        for node_id in node_ids:
+            subgraph.node_to_subgraph[
+                _get_dict_key(node_id)
+            ] = subgraph.node_to_subgraph[_get_dict_key(node_id)][
+                return_layers[0]
+            ]
+
```

Comment:
This looks like it would break existing access to this function. Could we add a parameter to the function call. Something like: `return_flattened` to trigger this?

Reply:
We could, but it's also nice for the API to be the same on v2 and v1, and I'd argue that returning an array on default is a little better than the existing behavior of by default returning a dict. But I'm fine with it either way -- if there are users running `cg.get_subgraph_nodes` on their own for which this change would break their code, maybe adding a parameter is better

Reply:
We already use `flattened` argument in various functions so I think its better to use it and let the user decide

Reply:
Sounds good. I added a keyword and set the default to False to not break existing access to the function.

Reply:
great!